src/main/java/bio/terra/app/controller/DataRepositoryServiceApiController.java

```diff
@@ -155,6 +156,13 @@ public ResponseEntity<DRSObject> getObject(
     return new ResponseEntity<>(drsObject, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<DRSAuthorizations> optionsObject(
+      @PathVariable("object_id") String objectId) {
```

Comment:
Our code does this everywhere, but I think we should get out of the habit of copying annotations down when implementing controller APIs. It has downsides and provides no benefit.
```suggestion
  public ResponseEntity<DRSAuthorizations> optionsObject(String objectId) {
```

Reply:
I took your suggestion, I'd be interested in hearing more about these downsides.

Reply:
Since the subclass annotation overrides the base class one, any changes made in the openapi yaml won't be reflected in the app until the subclass annotation is re-copied. The compiler and IDE don't provide warnings if the annotations differ (since it's not an error) so it's easy to think you've updated something when the change has no effect. The most common case I've seen this is when changing a default value, since those are things we don't always test.

That's a pretty big downside on its own; the others are minor -- having them clutters up the code, copying them down is an extra step, there are many annotations so how do we know which ones to copy, etc.

Reply:
This was really helpful, Phil.  Thanks!